frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -157,7 +160,11 @@ const PersonForm = (props: Props) => {
     setPatient(person);
     setAddressModalOpen(false);
     setFormChanged(false);
-    props.savePerson(person);
+    props.savePerson(
+      Object.fromEntries(
+        Object.entries(person).map(([key, value]) => [key, value || null])
```

Comment:
This is going to break for falsey values. For example `residentCongregateSetting` and `employedInHealthcare`. Values of `false` mean "No" for these questions but will be changed to `null`, which means "Unknown" (and will literally map to the string "Unknown" after my PR that's in flight.

I'm wondering what values _should_ be changed to `null` here. Probably just `undefined`? Would this make sense?

```javascript
Object.entries(person).map(([key, value]) => [key, value === undefined ? null : value])
```